<launch>
  <arg name="box_namespace" default="gt_box"/>
  <group ns="$(arg box_namespace)">  
    <node pkg="box_recording"
        name="rosbag_record_node_lpc"
        type="rosbag_record_node_grpc.py"
        output="screen">
        <param name="capability_group" value="Recording"/>
        <param name="port" value="50051"/>
        <param name="store_debug_logs" value="True"/>
    </node>
  </group>
</launch>